acquia/mc-cs-plugin-custom-objects
# Handling custom item links when merging contacts

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [y]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes https://acquia.atlassian.net/browse/MAUT-11209



#### Description:

If 2 contacts are merged into 1 then the links to custom items from the loser contacts were lost. This PR adds them to the victor contacts instead.

#### Steps to test this PR:


1. Create 2 contacts
2. Create at least one custom object
3. Create at least one custom item
4. Link the custom item with first contact
5. Merge the first contact into the second one
6. Check that the second contact have the link from the loser contact.
